strapi/strapi
# Fix primary button size on API Tokens List and Edit views

What does it do?
As part of the Design System we defined that in the header component, the primary button should be a S size, which is the case in the CM and in the CTB. However, in the API Tokens List and Edit views, we use a L size. 

How to test it?
Should satisfy the following criterias:

 - Change the primary button to a S size in both the API Tokens List and Edit views.
